src/test/java/use_cases/add_edit_category_use_case/CategoryUCITest.java

```diff
@@ -0,0 +1,56 @@
+package use_cases.add_edit_category_use_case;
+
+import entities.Category;
+import entities.EntityException;
+import entities.MonthlyStorage;
+import entities.SessionStorage;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import views.add_edit_category_views.CategoryP;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CategoryUCITest {
+
+
+    @Test
```

Comment:
It's unclear if you are testing success case or fail case. If testing success case, you should not use Assertions.fail()

Reply:
Yep, agree. This is the same thing I was doing in my test. If you are going to Assertions.fail() when an exception is caught, then you can just let that exception through so that the test fails anyways